pkg/cmd/dev/build.go

```diff
@@ -40,7 +41,7 @@ func makeBuildCmd(runE func(cmd *cobra.Command, args []string) error) *cobra.Com
 	buildCmd := &cobra.Command{
 		Use:   "build <binary>",
 		Short: "Build the specified binaries",
-		Long:  "Build the specified binaries.",
+		Long:  fmt.Sprintf("Build the specified binaries:\n\t%s", strings.Join(allBuildTargets, "\n\t")),
```

Comment:
Maybe worth mentioning that you can also pass normal Bazel targets in addition to these shorthands.

Reply:
Done.